File: tests/fixtures/javascript/embedded_block.js
function getActiveUsers(db) {
    const query = `
        SELECT u.id,
               u.first_name,
               u.last_name,
               u.email,
               u.phone,
               u.created_at,
               u.updated_at,
               u.last_login,
               u.status,
               u.role,
               u.department,
               u.manager_id,
               u.location,
               u.timezone,
               u.language
        FROM users u
        JOIN departments d ON u.department_id = d.id
        WHERE u.status = 'active'
          AND u.deleted_at IS NULL
        ORDER BY u.last_name
    `;
    return db.execute(query);
}

function simpleQuery(db) {
    return db.execute("SELECT id FROM users");
}

File: tests/fixtures/go/embedded_block.go
package main

import "fmt"

func RunMigration() string {
	query := `
		CREATE TABLE IF NOT EXISTS users (
			id         SERIAL PRIMARY KEY,
			username   VARCHAR(255) NOT NULL UNIQUE,
			email      VARCHAR(255) NOT NULL UNIQUE,
			password   VARCHAR(255) NOT NULL,
			first_name VARCHAR(100),
			last_name  VARCHAR(100),
			role       VARCHAR(50) DEFAULT 'user',
			active     BOOLEAN DEFAULT TRUE,
			created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
			updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
			last_login TIMESTAMP,
			avatar_url TEXT,
			bio        TEXT,
			settings   JSONB DEFAULT '{}',
			verified   BOOLEAN DEFAULT FALSE,
			deleted_at TIMESTAMP,
			org_id     INTEGER REFERENCES organizations(id)
		);
		CREATE INDEX idx_users_email ON users(email);
		CREATE INDEX idx_users_username ON users(username);
		CREATE INDEX idx_users_org ON users(org_id);
	`
	fmt.Println("Running migration")
	return query
}
